ERCS/erc-7802.md

```diff
@@ -0,0 +1,270 @@
+---
+eip: 7802
+title: Crosschain Token Interface
+description: Minimal token interface for cross-chain transfers
```

Comment:
> Minimal

Too many standards claim to be minimal. I'd recommend removing this, and using your description to further elaborate on the ideas introduced in your title.

Reply:
Modified this section, let me know your thoughts now